[2024-03-21 16:05:02,079] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: airflow_exam.retrieve_raw_data 2024-03-21T16:04:00+00:00 [queued]>
[2024-03-21 16:05:02,103] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: airflow_exam.retrieve_raw_data 2024-03-21T16:04:00+00:00 [queued]>
[2024-03-21 16:05:02,103] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2024-03-21 16:05:02,103] {taskinstance.py:1088} INFO - Starting attempt 1 of 6
[2024-03-21 16:05:02,103] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2024-03-21 16:05:02,117] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): retrieve_raw_data> on 2024-03-21T16:04:00+00:00
[2024-03-21 16:05:02,127] {standard_task_runner.py:52} INFO - Started process 5124 to run task
[2024-03-21 16:05:02,131] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', '***_exam', 'retrieve_raw_data', '2024-03-21T16:04:00+00:00', '--job-id', '9218', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/DAG.py', '--cfg-path', '/tmp/tmp2x80lkq0', '--error-file', '/tmp/tmpcnv7lk0a']
[2024-03-21 16:05:02,132] {standard_task_runner.py:77} INFO - Job 9218: Subtask retrieve_raw_data
[2024-03-21 16:05:02,184] {logging_mixin.py:104} INFO - Running <TaskInstance: ***_exam.retrieve_raw_data 2024-03-21T16:04:00+00:00 [running]> on host 3a3d0401ac26
[2024-03-21 16:05:02,246] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=***_exam
AIRFLOW_CTX_TASK_ID=retrieve_raw_data
AIRFLOW_CTX_EXECUTION_DATE=2024-03-21T16:04:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-03-21T16:04:00+00:00
[2024-03-21 16:05:03,043] {python.py:151} INFO - Done. Returned value was: None
[2024-03-21 16:05:03,056] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=***_exam, task_id=retrieve_raw_data, execution_date=20240321T160400, start_date=20240321T160502, end_date=20240321T160503
[2024-03-21 16:05:03,096] {taskinstance.py:1265} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2024-03-21 16:05:03,108] {local_task_job.py:151} INFO - Task exited with return code 0
